This patch is carried over from WRL4.3 to disable html docs

Upstream Status: Pending

Signed-of-by: Aws Ismail <aws.ismail@windriver.com>

diff --git a/configure.ac b/configure.ac
index d5f7812..f769e0c 100644
--- a/configure.ac
+++ b/configure.ac
@@ -336,7 +336,7 @@ AM_CONDITIONAL(BUILD_SOLARIS, test -n "${SOLARIS_OPTS}")
 AC_SUBST([DARWIN_OPTS])
 AC_SUBST([SOLARIS_OPTS])
 
-AM_CONDITIONAL(BUILD_HTML_DOCS, test -n "${GROFF}")
+AM_CONDITIONAL(BUILD_HTML_DOCS, 0)
 
 AC_SUBST([LINT_FLAGS])

